app/views/idv/shared/_document_capture.html.erb

```diff
@@ -142,7 +142,17 @@
 
   <p class='margin-top-4 margin-bottom-0'><%= t('doc_auth.info.document_capture_upload_image') %></p>
 </div>
-<%= render 'idv/doc_auth/start_over_or_cancel' %>
+
+<%= render 'idv/doc_auth/start_over' %>
+<% if session['idv/capture_doc'] %>
```

Comment:
An alternative could be to pass a boolean as a local, since the two places we're rendering the partial are for [`doc_auth`](https://github.com/18F/identity-idp/blob/master/app/views/idv/doc_auth/document_capture.html.erb) and [`capture_doc`](https://github.com/18F/identity-idp/blob/master/app/views/idv/capture_doc/document_capture.html.erb) steps, which we could pass `false` and `true` respectively. I imagine it might make it easier to add tests for the behavior as well.